install.sh

```diff
@@ -759,6 +759,13 @@ then
   additional_shellenv_commands+=("export HOMEBREW_CORE_GIT_REMOTE=\"${HOMEBREW_CORE_GIT_REMOTE}\"")
 fi
 
+if [[ -n "${HOMEBREW_INSTALL_FROM_API-}" ]]
+then
+  ohai "HOMEBREW_INSTALL_FROM_API is set."
+  echo "Homebrew/homebrew-core will not be tapped during this ${tty_bold}install${tty_reset} run."
+  additional_shellenv_commands+=("export HOMEBREW_INSTALL_FROM_API=\"${HOMEBREW_INSTALL_FROM_API}\"")
```

Comment:
If it's already set: why does it need exported?

Reply:
Thanks for the reminder, this is copied from the code block just above and indeed not needed (it was for non default remotes).